dev-util/diffball/diffball-1.0.1-r1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic
+
+DESCRIPTION="Delta compression suite for using/generating binary patches"
+HOMEPAGE="https://github.com/rafaelmartins/diffball"
+SRC_URI="https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/${PN}/${P}.tar.bz2"
+
+LICENSE="BSD"
```

Comment:
According to source code, it's a different license?

Reply:
It's BSD 3 clauses
https://github.com/rafaelmartins/diffball/commit/95c18477dea9c9dd2b362ff40e5a7762195e8d98#diff-7116ef0705885343c9e1b2171a06be0e

Reply:
Source code indicates ```GPL-2+```. Source code overrides ```COPYING```.
See https://devmanual.gentoo.org/general-concepts/licenses/index.html.